docs/source/conf.py

```diff
@@ -178,8 +178,8 @@ def _transform_changelog(path_in: str, path_out: str) -> None:
 # documentation.
 
 html_theme_options = {
-    'pytorch_project': info.__homepage__,
-    'canonical_url': info.__homepage__,
+    'pytorch_project': 'https://pytorchlightning.ai',
```

Comment:
why not keep `info.__homepage__` and update the link there?

Reply:
some SEO :]
cc: @aribornstein

Reply:
This points to the github and we want these urls to point to the site where the docs are hosted.